src/purchases.ts

```diff
@@ -187,7 +162,7 @@ export default class Purchases {
      * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults suite, otherwise it will use standardUserDefaults.
      * Default is null, which will make the SDK use standardUserDefaults.
      */
-    public static setup(
+    public static configure(
```

Comment:
I think we might wanna take the opportunity to also update configure to use the `setX` + `builder` pattern we've used for the others, since we're doing the rename for consistency

Reply:
Oh yeah let me do that

Reply:
I'll do that as a separate PR though.